crates/iceberg-catalog/src/service/storage/gcs/mod.rs

```diff
@@ -192,7 +192,7 @@ impl GcsProfile {
                         .duration_since(std::time::UNIX_EPOCH)
                         .unwrap()
                         .as_millis()
-                        + expiry as u128)
+                        + (expiry * 1000) as u128)
```

Comment:
fixes wrong expiry offset, expiry is expected to millis but returned as secs by gcs